common/src/main/java/muramasa/gregtech/gui/widgets/LavaBoilerWidget.java

```diff
@@ -0,0 +1,110 @@
+package muramasa.gregtech.gui.widgets;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import muramasa.antimatter.gui.GuiInstance;
+import muramasa.antimatter.gui.IGuiElement;
+import muramasa.antimatter.gui.Widget;
+import muramasa.antimatter.gui.container.ContainerMachine;
+import muramasa.antimatter.gui.widget.WidgetSupplier;
+import muramasa.antimatter.mixin.client.AbstractContainerScreenAccessor;
+import muramasa.gregtech.tile.single.TileEntityLavaBoiler;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
```

Comment:
in common folder but it uses fabricmc?

Reply:
This appears to be the case with the existing `CoalBoilerWidget` which I used as a reference to update the old PR code to fit into the current code base.

Reply:
Just checked and Antimatter API also uses the Fabric API `@Environment` annotation in a few places too.

Reply:
These 2

Reply:
I'll resolve this based on Trinsdar's comment.